### ISSUE TITLE
[Feature] Add the ability to add rows of items

### ISSUE THREAD

**Issue by akinsho** [goolord/alpha-nvim#187]
Hi 👋🏿,

Been using this plugin for a while and am very pleased with it, many thanks. I recently went down the path of trying to add my sessions to my dashboard (I have a lot) and one issue I've hit is that I can create columns of items so I'm constrained to vertical space but have plenty of free horizontal space.

<img width="1612" alt="Screenshot 2023-02-22 at 18 24 14" src="https://user-images.githubusercontent.com/22454918/220722709-ba8cbd58-7f6c-4e2d-be04-92a1f8569745.png">

As you can hopefully see from the screen shot since I want to keep the quote and my buttons I'm stuck with being only able to show a few session but there is so much horizontal space available that if I could create a row of session columns I could get probably two (or maybe a bit excessive three) columns in the top section.

WDYT?

NOTE: the sessions are interactive so I can do it by just drawing lines that happen to look like they are two separate columns I actually want to do a column of items

**Comment by goolord** [comment]
yeah this has been requested at least once before, I think it's somewhat easily doable. I'll try to get around to this

**Comment by ElPiloto** [comment]
Thanks for the awesome plugin! Any update on being able to add columns of content?

**Comment by blyedev** [comment]
Thanks for the awesome plugin, does it look like this feature will come along?

**Comment by a-clean-kitchen** [comment]
Bumping for interest
